### ISSUE TITLE
CC not installing on Ubuntu installed using UTM in Macbook Pro M1 (Arm based)

### ISSUE THREAD

**Issue by pavanhosatti** [CloudCompare/CloudCompare#1839]
I have been trying to install CloudCompare on my Ubuntu( which is installed using UTM in Macbook Pro M1 Arm based) and I am getting pending status in Software packages in Ubuntu. I tried installing using flatpak, but even that does not work as it tells me it could not find the cloudcompare software. The error is as follows:

"error: Nothing matches org.cloudcompare.CloudCompare in remote flathub."

**Comment by rjanvier** [comment]
seems an exotic setup :). Maybe It looks like a network issue inside your VM?

**Comment by pavanhosatti** [comment]
Hey, Network is fine in VM as i confirmed it by installing other applications.

**Comment by rjanvier** [comment]
Coult you please past here the flatpak command you use?

**Comment by pavanhosatti** [comment]
I used the following command:

"flatpak install flathub org.cloudcompare.CloudCompare"

**Comment by rjanvier** [comment]
yes but did you follow these steps? https://flathub.org/setup/Ubuntu

**Comment by pavanhosatti** [comment]
Yes, I followed all these steps.

**Comment by rjanvier** [comment]
flatpak and snap are not correlated, so it really looks like a network/firewall/proxy issue.

**Comment by pavanhosatti** [comment]
Hey, I have been able to install other applications using flatpak. I used the following command to install some application and it worked (App called Jami):

flatpak install flathub net.jami.Jami"

**Comment by rjanvier** [comment]
maybe you can try to install this one https://github.com/CloudCompare/org.cloudcompare.CloudCompare/releases/download/untagged-55ac1eee552f131ebf65/cloudcompare.flatpak . thanks

**Comment by pavanhosatti** [comment]
Hey, Is that the correct link? I am getting the 404 page not found error.

**Comment by rjanvier** [comment]
sure it is.

**Comment by pavanhosatti** [comment]
But, I am getting 404 Page not found error.

**Comment by rjanvier** [comment]
Hi, beta version is available on Flathub
Please read [this](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) if you want to know the basic about beta channel on flathub. For CC, basically you have to do:

flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.cloudcompare.CloudCompare

You may need to make the beta version the default one with

flatpak make-current org.cloudcompare.CloudCompare beta
